funding/manager.go

```diff
@@ -4136,6 +4136,28 @@ func (f *Manager) handleInitFundingMsg(msg *InitFundingMsg) {
 	// remote party.
 	chanReserve := f.cfg.RequiredRemoteChanReserve(capacity, ourDustLimit)
 
+	// Check the sanity of the selected channel constraints.
+	channelConstraints := &channeldb.ChannelConstraints{
+		DustLimit:        ourDustLimit,
+		ChanReserve:      chanReserve,
+		MaxPendingAmount: maxValue,
+		MinHTLC:          minHtlcIn,
+		MaxAcceptedHtlcs: maxHtlcs,
+		CsvDelay:         remoteCsvDelay,
+	}
+	err = lnwallet.VerifyConstraints(
+		channelConstraints, resCtx.maxLocalCsv, capacity,
+	)
+	if err != nil {
+		_, err := f.cancelReservationCtx(peerKey, chanID, false)
+		if err != nil {
+			log.Errorf("unable to cancel reservation: %v", err)
+		}
+
+		msg.Err <- err
+		return
```

Comment:
I think we need to return the error here as well.

Now when calling `./lncli-debug -n regtest openchannel --remote_reserve_sats=100 02d0830a548c670525ae0f17c8c2f2d9a3f0c520da0cc75591394198092e46ce1a 50000` I don't get any response in my terminal